pom.xml

```diff
@@ -72,11 +72,122 @@
+        <jsonPath.version>2.0.0</jsonPath.version>
+        <commons-compress.version>1.4</commons-compress.version>
+        <qpid.version>0.20</qpid.version>
+        <mongodb.version>3.0.3</mongodb.version>
+        <riak.version>1.4.0</riak.version>
+        <maven-war-plugin.version>2.4</maven-war-plugin.version>
+        <validation-api.version>1.0.0.GA</validation-api.version>
+        <geronimo-jms_1.1_spec.version>1.1.1</geronimo-jms_1.1_spec.version>
+        <sleepycat-je.version>5.0.34</sleepycat-je.version>
+        <org.marre.smsj.version>1.0.0-20051126</org.marre.smsj.version>
+        <mysql-connector-java.version>5.1.18</mysql-connector-java.version>
+        <hadoop.version>1.0.2</hadoop.version>
+        <commons-cli.version>1.2</commons-cli.version>
+        <postgresql.version>9.1-901.jdbc4</postgresql.version>
+        <activemq.version>5.10.0</activemq.version>
+        <rabbitmq-version>2.8.7</rabbitmq-version>
+        <kafka.version>0.8.2.1</kafka.version>
+        <storm.version>0.8.2</storm.version>
+        <redis.version>1.5.2</redis.version>
+        <astyanax.version>1.56.24</astyanax.version>
+        <jcouchdb.version>0.11.0-1</jcouchdb.version>
+        <solr.version>4.7.0</solr.version>
+        <jtidy.version>r8-20060801</jtidy.version>
+        <opendmk_jmxremote_optional_jar.version>1.0-b01-ea</opendmk_jmxremote_optional_jar.version>
+        <resteasy.version>3.0.8.Final</resteasy.version>
+        <maven-dependency-plugin.version>2.8</maven-dependency-plugin.version>
+        <jopt.version>4.3</jopt.version>
+        <concurrentlinkedhashmap.version>1.0_jdk5</concurrentlinkedhashmap.version>
+        <maven-replacer-plugin.version>1.5.2</maven-replacer-plugin.version>
+        <nodejs-maven-plugin.version>1.0.3</nodejs-maven-plugin.version>
+        <nodejs-maven-binaries.version>0.10.25</nodejs-maven-binaries.version>
+        <jasmine-maven-plugin.version>1.3.1.5</jasmine-maven-plugin.version>
+        <requirejs-maven-plugin.version>2.0.0</requirejs-maven-plugin.version>
+        <maven-antrun-plugin.version>1.7</maven-antrun-plugin.version>
+        <jetty-orbit-javax-servlet.version>3.0.0.v201112011016</jetty-orbit-javax-servlet.version>
+        <jcommander.version>1.27</jcommander.version>
+        <xml-apis.version>1.0.b2</xml-apis.version>
+        <jsr250-api.version>1.0</jsr250-api.version>
+        <guice.version>3.0</guice.version>
+        <javax-inject.version>1</javax-inject.version>
+        <aopalliance.version>1.0</aopalliance.version>
+        <commons-configuration.version>1.7</commons-configuration.version>
+        <commons-lang.version>2.4</commons-lang.version>
+        <hamcrest.version>1.1</hamcrest.version>
+        <jsr311-api.version>1.1.1</jsr311-api.version>
+        <maxmind.version>0.8.1</maxmind.version>
+        <jna.version>4.0.0</jna.version>
+        <winrm4j.version>0.1.0</winrm4j.version>
+        <coverage.target>${working.dir}</coverage.target>
```

Comment:
Good spot. Not needed. Also the version of antrun used is 1.8 in ./parent.pom.xml. Either needs to be removed, or changed to 1.8 and modify ./parent/pom.xml to use the version propert. Thanks @bostko .